docs/rbac.rst

```diff
@@ -0,0 +1,266 @@
+
+.. important::
+
+    User ``admin`` automatically gets all RBAC permissions.
+
+RPM plugin presents Role Based Access for these endpoints :
+
+* RPM Alternate Content Source (``/pulp/api/v3/acs/rpm/rpm/``)
+* RPM Distributions (``/pulp/api/v3/distributions/rpm/rpm/``)
+* RPM Publications (``/pulp/api/v3/publications/rpm/rpm/``)
+* RPM Remotes (``/pulp/api/v3/remotes/rpm/rpm/`` and ``/pulp/api/v3/remotes/rpm/uln``)
+* RPM Repository (``/pulp/api/v3/repositories/rpm/rpm/``)
+
+Default Roles
+-------------
+
+RPM plugin ships a few default roles described bellow.
+
+Viewer Role
+^^^^^^^^^^^
+
+This role contains only ``view`` permission for an `endpoint`. Using this permission you can control who can
+view specific content for example a repository.
+
+.. code-block:: shell
+   :caption: You can check the role by pulp-cli
+
+    pulp role show --name "rpm.rpmremote_viewer"
+
+.. code-block:: shell
+
+    {
+      "pulp_href": "/pulp/api/v3/roles/62baf888-09c2-4c18-8d3a-354d9c6bdb48/",
+      "pulp_created": "2022-04-07T12:41:11.065567Z",
+      "name": "rpm.rpmremote_viewer",
+      "description": null,
+      "permissions": [
+        "rpm.view_rpmremote"
+      ],
+      "locked": true
+    }
+
+Creator Role
+^^^^^^^^^^^^
+
+This role contains only ``add`` permission for a specific endpoint to allow a user to create a new object of its type.
+Be aware that you may need additional permission to create a new object.
+For example, ``rpm.rpmrepository_view`` is needed for a used repository to create a new publication.
+
+.. code-block:: shell
+   :caption: Example of the requested condition to create an RPM Publication
+
+    {
+        "action": ["create"],
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": [
+            "has_model_perms:rpm.add_rpmpublication",
+            "has_repo_attr_model_or_obj_perms:rpm.view_rpmrepository",
+        ]
+    }
+
+Owner Role
+^^^^^^^^^^
+
+This role contains all permissions needed for manipulating objects, such as modifying or removing them.
+This role also contains permission to manage role assignments on the object,
+for example, adding or removing user roles.
+
+By default, the user who created a new object will receive this role.
+
+.. code-block:: shell
+   :caption: Example of the RPM Remote owner role
+
+    {
+      "pulp_href": "/pulp/api/v3/roles/592356da-8cdf-4a1b-a434-3f10cda285f7/",
+      "pulp_created": "2022-04-07T12:41:11.047168Z",
+      "name": "rpm.rpmremote_owner",
+      "description": null,
+      "permissions": [
+        "rpm.change_rpmremote",
+        "rpm.delete_rpmremote",
+        "rpm.manage_roles_rpmremote",
+        "rpm.view_rpmremote"
+      ],
+      "locked": true
+    }
+
+RPM specific Roles
+------------------
+
+Also, the RPM plugin ships two roles to use.
+
+RPM Admin Role (rpm.admin)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This role is like a pulp admin but only for the RPM plugin.
+You can perform every RPM workflow. If you need a more
```

Comment:
This role is like a pulp admin but only for the RPM plugin - it can perform any RPM workflow with no restrictions.